Resources/Prototypes/_SpaceCats/Entities/Clothing/Uniforms/jumpsuits.yml

```
@@ -0,0 +1,53 @@
+  - type: Sprite
+    sprite: _SpaceCats/Clothing/Uniforms/Jumpsuit/captain_command.rsi
+  - type: Clothing
+    sprite: _SpaceCats/Clothing/Uniforms/Jumpsuit/captain_command.rsi
+
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpsuitWhiteCaptain
+  name: white captain jumpsuit
+  description: Captain white jumpsuit symbolizes that your salary is clearly above the norm.
+  components:
+    - type: Sprite
```

yamllint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)